explainability-connectors/src/main/java/org/kie/trustyai/connectors/kserve/v2/PayloadParser.java

```diff
@@ -12,9 +12,15 @@
 public class PayloadParser {
 
     public static PredictionInput inputTensorToPredictionInput(ModelInferRequest.InferInputTensor tensor,
-            List<String> inputNames) {
+            List<String> inputNames) throws IllegalArgumentException {
         final InferTensorContents responseInputContents = tensor.getContents();
-        final KServeDatatype type = KServeDatatype.valueOf(tensor.getDatatype());
+        final KServeDatatype type;
+        try {
+            System.out.println(tensor.getDatatype());
```

Comment:
To be replaced by `log` or removed?

Reply:
Removed :) Good catch, thanks @danielezonca